modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -134,6 +136,11 @@ const factory: UiSketcher.CompositeSketchFactory<OuterContainerSketchDetail, Out
         Optionals.mapFrom(toolbar.getApis<ToolbarApis>().toggle, (toggle) => toggle(toolbar));
       });
     },
+    toggleToolbarDrawerWithoutFocusing: (comp) => {
+      Composite.parts.getPart(comp, detail, 'toolbar').each((toolbar) => {
+        Optionals.mapFrom(toolbar.getApis<ToolbarApis>().toggleWithoutFocusing, (toggleWithoutFocusing) => toggleWithoutFocusing(toolbar));
```

Comment:
Generally, you would always use the Sketcher API directly, like 

```
SplitFloatingToolbar.toggleWithoutFocusing
```

So this could become 

```
Composite.parts.getPart(comp, detail, 'toolbar').each(SplitFloatingToolbar.toggleWithoutFocusing)
```

Is the reason you avoided using that you wanted to handle when `toggleWithoutFocusing` was not defined because the `toolbar` part component wasn't a `SplitFloatingToolbar` ?

Reply:
> Is the reason you avoided using that you wanted to handle when `toggleWithoutFocusing` was not defined because the `toolbar` part component wasn't a `SplitFloatingToolbar` ?

Yes, I've followed the `toggleToolbarDrawer` function example.